mod/projects/set/type_plus_right/pointer/project.rb

```diff
@@ -4,7 +4,7 @@
 include_set Type::SearchType
 
 def virtual?
-  true
+  new?
```

Comment:
I think this is a better practice than `true`. It avoids ever returning true when there is a real card.